src/scheduler/server_info.c

```diff
@@ -223,8 +223,7 @@ query_server(status *pol, int pbs_sd)
 		return NULL;
 	}
 
-
-	if ((sched = pbs_statsched(pbs_sd, PBS_DFLT_SCHED_NAME, NULL, NULL)) == NULL) {
+	if ((sched = pbs_statsched(pbs_sd, sc_name, NULL, NULL)) == NULL) {
```

Comment:
This change will conflict with PR #559. I suggest we hold off reviews until the other PR is merged, which should happen in the next few hours. Then you may rebase your branch and resolve the conflicts.

Reply:
PR #559  has been merged. Please rebase your branch.